src/main/java/com/conveyal/gtfs/graphql/fetchers/PolylineFetcher.java

```diff
@@ -21,6 +21,15 @@
 
 /**
  * GraphQL fetcher to get encoded polylines for all shapes in a GTFS feed.
+ *
+ * Note: this fetcher was developed to prevent out of memory errors associated with requesting all trip patterns with
+ * their associated shapes. Previously, attempting to fetch all shapes for a large feed would require a separate SQL
+ * query for each shape (to join on shape_id) and result in many duplicated shapes in the response.
+ *
+ * Running a shapes polyline query on a 2017 Macbook Pro (2.5 GHz Dual-Core Intel Core i7) results in the following:
+ * - NL feed (11291 shapes): ~12 seconds (but nobody should be editing such a large feed in the editor)
+ * - TriMet (1302 shapes): ~8 seconds
+ * - MBTA (1272 shapes): ~4 seconds
```

Comment:
Good to know, but a response time of 4 seconds is definitely something that could be improved upon.